pkg/applicationmanager/controllers/applicationbackupschedule.go

```diff
@@ -101,11 +103,77 @@ func (s *ApplicationBackupScheduleController) Reconcile(ctx context.Context, req
 }
 
 // Handle updates for ApplicationBackupSchedule objects
-func (s *ApplicationBackupScheduleController) handle(ctx context.Context, backupSchedule *stork_api.ApplicationBackupSchedule) error {
+func (s *ApplicationBackupScheduleController) handle(_ context.Context, backupSchedule *stork_api.ApplicationBackupSchedule) error {
 	if backupSchedule.DeletionTimestamp != nil {
 		return nil
 	}
 
+	// Check whether if VolumeSnapshotClassName is given. If yes, check it's using the older way of requestParams. If yes, then migrate
+	// to new map in case of csi based backups
+	if backupSchedule.Spec.Template.Spec.Options != nil {
```

Comment:
Since the set of code is used in both files. Can this be merged to one function under utils and called from both backup and backupschedule?

Reply:
@vsundarraj-px both code looks similar but have different object and their structure, also it has different loggers and events

Reply:
You can fetch the result from the API and update the object outside of the API. 
Please avoid repeat code.